pkg/kev/kev_reconcile_test.go

```diff
@@ -528,12 +532,13 @@ var _ = Describe("Reconcile", func() {
 
 func newDefaultServiceExtensions(name string, k8sconfs ...config.K8SConfiguration) (map[string]interface{}, error) {
 	k8s := config.K8SConfiguration{
-		Enabled: true,
+		Disabled: false,
 		Workload: config.Workload{
 			LivenessProbe:  config.DefaultLivenessProbe(),
 			ReadinessProbe: config.DefaultReadinessProbe(),
 			Type:           config.DefaultWorkload,
 			Replicas:       config.DefaultReplicaNumber,
+			RestartPolicy:  "always",
```

Comment:
This should be [config.DefaultRestartPolicy](https://github.com/appvia/kev/blob/master/pkg/kev/config/defaults.go#L29-L30) I reckon.

Reply:
config.DefaultRestartPolicy is "Always" and the specific code from project_service was setting "always" so i left it as it was to avoid adding to the changes. It's definitely something that can be improved later